free-2.md

```diff
@@ -11,41 +11,88 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+There is a need to compile and link C/C++ files by the software development community.
+Currently, the common way of compiling and linking C/C++ code is through a DSL called
+make. However, make is very very confusing to use and doesnt seem to have much fluency
+so I think it really needs a revamp. If i could help them, their experience would be
+much less confusing. I imagine they could easily just define the dependencies in a more
+natural way than the Syntactic-Aspartame filled make. 
```

Comment:
Have you seen [CMake](https://cmake.org) or [SCons](http://scons.org)? Both are more portable and usable than make, and CMake specifically is designed based on make